cpp/src/tests/sssp/.clang-format

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
I am not against adding a clang format file, but I missed the discussion on setting these parameters. Is this something standard that we are using elsewhere?

Reply:
This was committed accidentally. It should have been in gitignore. That said, I just picked the params after some basic trial and error. Also, I didn't attempt any deeper integration with clang's tools. This was just a quick fix to get some basic formatting.

Reply:
Let's remove it for now and address this as part of another PR then

Reply:
https://github.com/rapidsai/cugraph/issues/326